base-math/truncdddf.c

```diff
@@ -49,7 +52,20 @@ CONVERT_WRAPPER(
 	  {
 	    if (DFP_EXCEPTIONS_ENABLED)
 	      DFP_HANDLE_EXCEPTIONS (FE_OVERFLOW|FE_INEXACT);
-	    return SIGNBIT(a) ? -INFINITY : INFINITY;
+
+		/* Default result when trap is disable - IEEE754 */
+	    switch (fegetround())
+	      {
+	        case FE_TOWARDZERO:
+	          return SIGNBIT(a) ? -DBL_MAX : DBL_MAX;
+	        case FE_DOWNWARD:
+	          return SIGNBIT(a) ? -INFINITY : DBL_MAX;
+	        case FE_UPWARD:
+	          return SIGNBIT(a) ? -DBL_MAX : INFINITY;
+	        case FE_TONEAREST:
+	        default:
+	          return SIGNBIT(a) ? -INFINITY : INFINITY;
```

Comment:
In lines 59 to 67, replace 8 spaces with 1 tab.

Reply:
@gftg85 I was following how it's implemented on the rest of the file. On my vim it appears (set list) as one tab and spaces. But ok I'll check that.

Reply:
Hrm, you are right.  Please disregard this comment.

It seems that there isn't a coding style being followed and the relevant section in README.developer is in TODO state.

Sorry about the inconvenience.